lib/rubygems.rb

```diff
@@ -1325,22 +1325,6 @@ def default_gem_load_paths
 require_relative 'rubygems/exceptions'
 
 # REFACTOR: This should be pulled out into some kind of hacks file.
```

Comment:
I think the comment references the code block you moved.

Reply:
Yeah, I think it references both the block I moved and the block underneath (loading implementation specific hacks). And kernel monkeypatches below are also hacks, so I figured I would leave the comment there, implying "everything from  now on is a hack" :sweat_smile: